examples/jsm/nodes/accessors/ModelViewProjectionNode.js

```diff
@@ -4,20 +4,29 @@ import { cameraProjectionMatrix } from './CameraNode.js';
 import { modelViewMatrix } from './ModelNode.js';
 import { positionLocal } from './PositionNode.js';
 import { nodeProxy } from '../shadernode/ShaderNode.js';
+import { varying } from '../core/VaryingNode.js';
 
 class ModelViewProjectionNode extends TempNode {
 
-	constructor( positionNode = positionLocal ) {
+	constructor( positionNode = null ) {
 
 		super( 'vec4' );
 
 		this.positionNode = positionNode;
 
 	}
 
-	setup() {
+	setup( builder ) {
 
-		return cameraProjectionMatrix.mul( modelViewMatrix ).mul( this.positionNode );
+		if ( builder.shaderStage === 'fragment' ) {
+
+			return varying( builder.context.mvp );
```

Comment:
This seems like a bit of a hack to me -- I propose instead to introduce a property like `vertexOutput` which can be used here (and remove `builder.context.mvp` then). @sunag What do you think?